plugins/org.eclipse.glsp.api/src/org/eclipse/glsp/api/action/kind/DisposeClientAction.java

```diff
@@ -1,5 +1,5 @@
-/*******************************************************************************
- * Copyright (c) 2019 EclipseSource and others.
+/********************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
```

Comment:
In general you are right. But you picked a bad example. The DisposeClientAction was introduced with this commit so the copyright year should be 2020. This is displayed weirdly because Github is diffing GLSPClientAware <-> DisposeClientAction

Reply:
I did not touch any existing copyright headers in this PR